src/components/MapDisplay.tsx

```diff
@@ -5,24 +5,34 @@ import './MapDisplay.css'
 
 const LIGHT_RED = chart_global_danger_Color_100.value
 
+const mapsList: { [id: string]: mapboxgl.Map } = {}
```

Comment:
can we have type for `{ [id: string]: mapboxgl.Map }`

Reply:
Added.